server-connection/src/test/java/org/sonarsource/sonarlint/core/serverconnection/ServerPathProviderTests.java

```diff
@@ -54,4 +60,15 @@ void profile_token_generation_path_for_version_lower_than_9_7() {
     assertThat(serverPath).isEqualTo("baseUrl/account/security");
   }
 
+  @Test
+  void test() throws ExecutionException, InterruptedException {
```

Comment:
Maybe try to find a more descriptive name

Reply:
Sorry forgot to rename :)